chainsail_helpers/chainsail_helpers/pdf/__init__.py

```diff
@@ -1,14 +1,16 @@
 """
 Abstract PDF interfaces for Chainsail
 """
+from abc import abstractmethod, ABCMeta
 
 
-class PDF:
+class PDF(ABCMeta):
```

Comment:
When creating an ABC via inheritance, use `ABC` class as a parent class, see https://docs.python.org/3/library/abc.html#abc.ABC.
If you want to use `ABCMeta` you need to use it as a metaclass, like `PDF(metaclass=ABCMeta)`.

Reply:
Ah right, thanks a lot! Fixed in https://github.com/tweag/chainsail-resources/commit/982a4ca.